website/docs/running.md

```diff
@@ -123,7 +123,113 @@ podman run -p 8080:8080 --rm \
 ```
 
 ## Running with Operator
-- AeroGear 10145
 
+_UnifiedPush_ can be easily installed and maintained on OpenShift by using the _UnifiedPush Operator_. This Operator makes managing the UnifiedPush Server and Database a seamless process.There are a number of prerequisites to getting started with the UnifiedPush Operator.
+
```

Comment:
This is only necessary for development.  If you don't want to use the compiler and friends you can just apply the yaml files in the deploy directoy, and the crd files in deploy/crd.

Reply:
These could be moved to a section on development.